src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int const_11 = 11;
+	private static final int conts_minus1 = -1;
+	private static final int const_12 = 12;
 
-	public int total = -1;
+	final private int[] numbers = new int[const_12];
+	public int total = conts_minus1;
 
 	public void countIn(int in) {
```

Comment:
w sumie możnaby dać tutaj `final int in` gdyż nie zmieniamy tego parametru nigdzie